packages/terra-show-hide/src/_ShowHideButton.module.scss

```diff
@@ -0,0 +1,102 @@
+  input[type='reset'].button,
+  input[type='submit'].button {
+    appearance: button;  // autoprefix fix into -webkit-appearance
+  }
+  // stylelint-enable selector-no-qualifying-type
+
+  .button {
+    background-color: transparent; // Remove the gray background on active links in IE 10
+    border: 0;
+    color: var(--terra-show-hide-color, #0059a8);
+    cursor: pointer;
+    display: inline-block; // prevents variant icon from breaking onto its own line
+    font-family: inherit;
+    font-size: var(--terra-show-hide-font-size, 1rem);
+    line-height: var(--terra-show-hide-line-height, 1.429);
+    outline: none;
+    padding: 0;
+    text-align: center;
+    text-decoration: var(--terra-show-hide-underline, none);
+    text-transform: none;
+    touch-action: manipulation; // Enable fast tap interaction in webkit browsers; see https://webkit.org/blog/5610/more-responsive-tapping-on-ios/
+    user-select: none; // Prevent text selection on buttons on mobile devices
+    white-space: normal;
+    // Remove the inner border and padding in Firefox.
+    &::-moz-focus-inner {
+      border: 0;
+      padding: 0;
+    }
+
+    &::after {
+      background-image: var(--terra-show-hide-icon, inline-svg('<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" fill="#0059a8"><path d="M48 12L24 36 0 12h48z"/></svg>'));
+      background-repeat: no-repeat;
+      background-size: cover;
+      content: '';
+      display: inline-block;
+      height: var(--terra-show-hide-height, 0.6em);
+      margin-left: var(--terra-show-hide-margin-left, 0.4em);
+      width: var(--terra-show-hide-width, 0.6em);
+    }
+
+    &.is-focused {
+      background-color: var(--terra-show-hide-focus-background-color, transparent);
+      border-radius: var(--terra-show-hide-focus-border-radius, 0.2142rem);
+      box-shadow: var(--terra-show-hide-focus-box-shadow, 0 0 0 1px #4cb2e9, 0 0 0 3px rgba(76, 178, 233, 0.25));
+      color: var(--terra-show-hide-focus-color, #001f67);
+      text-decoration: var(--terra-show-hide-focus-underline, underline);
+
+      &::after {
+        background-image: var(--terra-show-hide-focus-icon, inline-svg('<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" fill="#001f67"><path d="M48 12L24 36 0 12h48z"/></svg>'));
+      }
+    }
+
+    &:hover,
+    &.is-hovered {
+      background-color: var(--terra-show-hide-hover-background-color);
+      color: var(--terra-show-hide-hover-color, #001f67);
+      text-decoration: var(--terra-show-hide-hover-underline, underline);
+
+      &::after {
+        background-image: var(--terra-show-hide-hover-icon, inline-svg('<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" fill="#001f67"><path d="M48 12L24 36 0 12h48z"/></svg>'));
+      }
+    }
+
+    &:active {
+      color: var(--terra-show-hide-active-color, #001f67);
+    }
+
+    &:disabled,
+    &.is-disabled {
+      color: var(--terra-show-hide-disabled-color, rgba(28, 31, 33, 0.2));
+      text-decoration: none;
```

Comment:
add to disabled:
```
pointer-events: none;
```

Reply:
The second commit removes a lint warning 
https://github.com/cerner/terra-core/pull/1956/commits/186d864902a97d95e46b0843754019af99d97dd5
https://github.com/cerner/terra-core/pull/1956/commits/635658991991f1bd4dff78f75704f8895cdc050d

Reply:
missed the opening tag '<' https://github.com/cerner/terra-core/pull/1956/commits/e123fa7133074b347053c4f622987b49e754cee6